api/src/routes/webhooks.ts

```
@@ -0,0 +1,188 @@
+  const discordWebhookUrl = `https://discord.com/api/webhooks/${id}/${token}`;
+
+  try {
+    // Parse incoming payload
+    const payload = await c.req.json();
+
+    // Transform payload based on provider
+    let discordPayload;
+
+    switch (provider.toLowerCase()) {
+      case 'doppler':
+        const result = dopplerSchema.safeParse(payload);
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)